modules/Microsoft.Network/networkManagers/securityAdminConfigurations/deploy.bicep

```diff
@@ -0,0 +1,67 @@
+@sys.description('Conditional. The name of the parent network manager. Required if the template is used in a standalone deployment.')
+param networkManagerName string
+
+@maxLength(64)
+@sys.description('Required. The name of the security admin configuration.')
+param name string
+
+@maxLength(500)
+@sys.description('Optional. A description of the security admin configuration.')
+param description string = ''
+
+@sys.description('Required. Enum list of network intent policy based services. String array containing any of: "All", "AllowRulesOnly" or "None". Default is "None".')
+param applyOnNetworkIntentPolicyBasedServices array = [ 'None' ]
```

Comment:
Instead of having it in the description I'd recommend an allowed set

Reply:
Agreed. @AlexanderSehr Updated.

Reply:
Thanks. Now let's also remove the default in the description 😄

Reply:
done :)